compiler/ballerina-compiler-api/src/main/java/org/ballerina/compiler/api/symbols/Qualifiers.java

```diff
@@ -22,15 +22,15 @@
  * 
  * @since 2.0.0
  */
-public enum Qualifier {
+public enum Qualifiers {
```

Comment:
This should probably remain a singular name I think. Looks sorta weird when used in APIs otherwise. e.g., `List<Qualifier>` reads as a list of qualifiers whereas `List<Qualifiers>` sorta gives the impression that it's a list of list of qualifiers. We've used the singular form in enums in other places. e.g., `Flag`

Reply:
I see. Ack. Updated the PR.